src/Components/InventoryPage/__tests__/__snapshots__/InventoryPage.test.js.snap

```diff
@@ -0,0 +1,11 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`InventoryTable should render correctly 1`] = `
```

Comment:
is the '1' in the name important? .. I have no idea how snapshots work

Reply:
It is auto generated file, so I didn't touch it :smile: Maybe it can contain more snapshots in one file for that one test and this '1' is ID :+1: